image/mkosi.reposdir/amzn2-core.repo

```diff
@@ -0,0 +1,39 @@
+[amzn2-core]
+name=Amazon Linux 2 core repository
+#mirrorlist=$awsproto://$amazonlinux.$awsregion.$awsdomain/$releasever/$product/$target/$basearch/mirror.list
+mirrorlist=https://amazonlinux-2-repos-us-east-2.s3.dualstack.us-east-2.amazonaws.com/2/core/latest/x86_64/mirror.list
+priority=10
+gpgcheck=1
+#gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-amazon-linux-2
+gpgkey=https://raw.githubusercontent.com/xsuchy/distribution-gpg-keys/main/keys/amazon-linux/RPM-GPG-KEY-amazon-linux-2
```

Comment:
```suggestion
gpgkey=https://cdn.amazonlinux.com/_assets/11CF1F95C87F5B1A.asc
```
Any reasons for not using the AWS source?

Reply:
THX :)